godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -76,5 +76,8 @@
   </include>
 
   <node pkg="tf" type="static_transform_publisher" name="ensenso_static_tf" args="0 0 0 0 1.57 0 kinect2_move_frame camera_optical_frame 100" />
+  <rosparam command="load" file="$(find meshing_plugins)/config/meshing.yaml" />
+  <rosparam command="load" file="$(find path_planning_plugins)/config/path_planning.yaml" />
+  <rosparam command="load" file="$(find godel_process_planning)/config/process_planning.yaml" />
```

Comment:
The idea was primarily that individual implementations of specific plugins might require different parameters. Keeping the configuration file in the folder where the plugin lives encapsulates the plugin and makes it easier to maintain.

Wrt process planning, it was just a matter of convenience. I'm fine with moving it around.